<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGuiRemoteControlClass</class>
 <widget class="QMainWindow" name="QtGuiRemoteControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGuiRemoteControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_1" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>340</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_2" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_3" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>400</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_4" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>430</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_5" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="LightButton" name="lightButton_6" native="true">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>480</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>176</y>
      <width>317</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QCheckBox" name="TotalWificheckBox1">
       <property name="text">
        <string>wifi总开关</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="TotalWificheckBox2">
       <property name="text">
        <string>前50个wifi开关</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="TotalWificheckBox3">
       <property name="text">
        <string>前100个wifi开关</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>343</y>
      <width>302</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_1">
         <property name="text">
          <string>手机1wifi开关</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_2">
         <property name="text">
          <string>手机2wifi开关</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_3">
         <property name="text">
          <string>手机3wifi开关</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_4">
         <property name="text">
          <string>手机4wifi开关</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_5">
         <property name="text">
          <string>手机5wifi开关</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="TotalWificheckBox_6">
         <property name="text">
          <string>手机6wifi开关</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777212</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>下载速率</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_1"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_5"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_6"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="SwitchControl" name="Switch" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LightButton</class>
   <extends>QWidget</extends>
   <header>lightbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SwitchControl</class>
   <extends>QWidget</extends>
   <header>SwitchControl.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtGuiRemoteControl.qrc"/>
 </resources>
 <connections/>
</ui>
